static/src/js/components/SpatialSelection/SpatialSelection.jsx

```diff
@@ -536,13 +536,13 @@ const SpatialSelection = (props) => {
       const buttons = drawToolbar.querySelectorAll('.leaflet-draw-draw-circle, .leaflet-draw-draw-polygon, .leaflet-draw-draw-rectangle, .leaflet-draw-draw-marker')
       buttons.forEach((button) => {
         if (button.classList.contains('leaflet-draw-draw-circle')) {
-          button.setAttribute('aria-label', 'Draw a spatial circle')
+          button.setAttribute('aria-label', 'Draw a spatial circle on the map to select a spatial extent')
```

Comment:
This should say "Draw a circle..." not "Draw a spatial circle...". Same for all the other methods.